src/providers/cortex/trulens/providers/cortex/provider.py

```diff
@@ -15,15 +15,15 @@ class Cortex(
 
     DEFAULT_MODEL_ENGINE: ClassVar[str] = "snowflake-arctic"
 
-    connection_parameters: Dict
+    connection_parameters: Any
```

Comment:
How about Dict[str, Any] or the worst case `dict` (which is what I'm using in my onging PR to unblock)

Reply:
Pydantic is detecting a dict with the RSA type value as PydanticUndefined rather than Dict/dict, so either of these fail

Reply:
ah I see :/

Reply:
Approving the PR now but tagging Piotr @piotrm0 to see if you have seen this issue and might have some suggestions here for the typing. (I saw similar pydantic issue on my branch recently)